mlir/utils/jenkins/Jenkinsfile

```diff
@@ -306,6 +322,9 @@ pipeline {
                      description: 'Run the static library stage')
         booleanParam(name: 'perfTest', defaultValue: true,
                      description: 'Run the performance testing stage')
+        //TODO(@manupak): Switch the default to True after testing
```

Comment:
I'd just add this to, say, `weekly`

Reply:
same concern as above but I'd like a some subset (that we can bikeshed) to run on a PR basis